### DISCUSSION TITLE
Wiki.js has installed but menu items not rendered correctly

### DISCUSSION THREAD

**Discussion by pjamesmorgan** [requarks/wiki#3453] [Help / Questions]
OS: Windows Server 2016 Standard
Wiki.js version: 2.5.144
Node.js version: 12.18.4
PostgreSQL version: 12.4

I have installed Wiki.js as per instructions.  I configured it as per instructions and runs correctly on port 3000.  It starts up correctly, recognizing config.yml file and it connects to wiki database without issue.

However menu items and heading don't appear correctly.  I am seeing menu items called  "dashboard.title" "groups.title" and items called "dashboard.lastlogins".  The exit button appears as "actions.exit".  These displayed items do not match screenshots I have seen online.  Any ideas on how to put this right.  I am under pressure to deliver a departmental wiki for my colleagues.

Kind regards

Peter Morgan

**Comment by mikecouchman** [comment]
You need to get the locales installed. Does your server have internet access (no user/pass required)? If not, you need to download the locales .json file and sideload them.

**Comment by mikecouchman** [comment]
#2419 is the reference issue.

**Comment by pjamesmorgan** [comment]
Hi

I placed the locale files into the sideload folder and restarted Wiki.js.  Nothing happened.

Regards

Peter

Get Outlook for Android<https://aka.ms/ghei36>

Sent from my Huawei P20 Pro

________________________________

**Comment by mikecouchman** [comment]
#2135 is this the rendered output you get? Text items show as placeholders? When you downloaded the locale files, did you download the .zip file, or each .json file? Some browsers will download the .json as html - if you open one of the .json files in notepad, does it look like JSON or HTML? Download the .zip file containing all the locales, then copy across the 2 you need. On your server, does the account running the process have WRITE access to the \data folder? And after restating the node process, try opening wiki in incognito mode (bypasses the cache).

**Comment by NaruZosa** [comment]
Encountering the same issue. Server has internet access.

Running in a docker container, tried both ghcr.io/linuxserver/wikijs and requarks/wiki:2 images.

Edit: Deleted data folder, restarted a couple of times then waited and it's now working. Using linuxserver image.

**Comment by gunnicom** [comment]
I searched about an hour, to finally find out that I did not set "offline: true" in the config.yml.
Maybe it would be good to add that to the sideload documentation.

**Comment by Freundschaft** [comment]
> ghcr.io/linuxserver/wikijs

I cant get it to work, also from inside the container I cant curl, keeps telling me hostnname is unknown even though nslookup is wokring
